CONTRIBUTING.rst

```diff
@@ -19,6 +19,12 @@ especially the
 `coding conventions <https://cosmo-epfl.github.io/librascal/dev_guide/coding-convention.html>`_
 for C++ code (Python code generally follows PEP 8).
 
+We now have a pull request template; please use it (especially for new
```

Comment:
we now have => we have

Reply:
meh, I'll change it once it's been around for a few weeks and everyone's gotten used to it.

Reply:
It's one word in a fairly visible file. We'll get it in our next cleanup action.